pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -360,3 +363,246 @@ def matrix_multiply_parallel(matrix_1, matrix_2, num_threads):
                                           i, j).result()
 
     return C
+
+
+
+
+
+
+def _get_value_opt_group(matrix, lookup_index: List[int]):
+    """
+    gets a value
+    """
+    return matrix[lookup_index[0]][lookup_index[1]]
```

Comment:
One liner functions aren't recommended. They just slow down the code.

Reply:
See my suggestions at https://github.com/codezonediitj/pydatastructs/pull/272/files#r426157438
You have to do something similar with every one liner function.